internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/endToEnd/TestActiveActiveIngestion.java

```diff
@@ -181,7 +181,7 @@ public void testLeaderLagWithIgnoredData() throws Exception {
     String keySchemaStr = recordSchema.getField(DEFAULT_KEY_FIELD_PROP).schema().toString();
     String valueSchemaStr = recordSchema.getField(DEFAULT_VALUE_FIELD_PROP).schema().toString();
     UpdateStoreQueryParams storeParms = new UpdateStoreQueryParams().setActiveActiveReplicationEnabled(true)
-        .setHybridRewindSeconds(360)
+        .setHybridRewindSeconds(604800) // 7 days
```

Comment:
Is this change intended with the Marker record change?

Reply:
Ya, we don't need it. Reverted. Thanks